core/tests/ui/desktop/e2e/account.spec.ts

```diff
@@ -85,7 +85,7 @@ test('Add and remove new address', async ({ page }) => {
 
   await page.getByRole('textbox', { name: 'Suburb/City Required' }).fill(city);
   await page.getByRole('combobox', { name: 'Choose state or province' }).click();
-  await page.getByLabel(state).click();
+  await page.getByLabel(state, { exact: true }).click();
```

Comment:
Test fails to identify strict locator when Kansas is the selected state since`Kansas` and `Arkansas` match for this selector.